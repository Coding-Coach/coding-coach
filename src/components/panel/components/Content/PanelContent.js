import React from 'react';
import styles from './panelcontent.scss';

const Content = ({ children }) => {
  return <section className={styles.mainContent}>{children}</section>;
};

export default Content;
