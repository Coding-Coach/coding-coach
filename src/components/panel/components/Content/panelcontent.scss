@import '~theme/base/variables';

.mainContent{
  margin-bottom: 20px;
  padding: 20px 10px;
}