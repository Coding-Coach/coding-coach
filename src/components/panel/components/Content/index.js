import PanelContent from './PanelContent';

export default PanelContent;
