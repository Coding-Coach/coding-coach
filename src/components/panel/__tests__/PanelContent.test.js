import React from 'react';
import { render } from 'react-testing-library';
import Content from '../PanelContent';

test('renders the PanelContent component', () => {
  const { container } = render(<Content />);

  expect(container.firstChild).toMatchSnapshot();

  const section = container.querySelectorAll('section');
  expect(section.length).toEqual(1);
});
