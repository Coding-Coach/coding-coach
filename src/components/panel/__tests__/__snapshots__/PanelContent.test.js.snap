// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the PanelContent component 1`] = `<section />`;
